core/common/src/main/scala/org/typelevel/otel4s/Attribute.scala

```diff
@@ -155,6 +232,9 @@ object Attribute {
   ): Attribute[Key] =
     Impl(select.make(name), from(value))
 
+  implicit def materializeFromKeyed[A, Key](value: A)(implicit ev: Keyed[A, Key]): Attribute[Key] =
```

Comment:
this is the part I don't really like. I think we should have them be constructed explicitly

Reply:
`Attribute.keyed(userId)`?